packages/reactor-api/src/processors/manager.ts

```diff
@@ -1,16 +1,16 @@
-import { IAnalyticsStore } from "@powerhousedao/analytics-engine-core";
 import { IDocumentDriveServer } from "document-drive";
 import { DocumentDriveDocument } from "document-model-libs/document-drive";
-import { IProcessor, IProcessorManager, ProcessorSetupArgs } from "./types";
-import { ProcessorClass, isProcessorClass } from "./processors";
+import { Knex } from "knex";
+import { ProcessorClass, isProcessorClass } from ".";
+import { IProcessor, IProcessorManager, ProcessorSetupArgs } from "../types";
 
 export class ProcessorManager implements IProcessorManager {
   private reactor: IDocumentDriveServer;
   private processors: IProcessor[] = [];
 
   constructor(
     driveServer: IDocumentDriveServer,
-    private analyticsStore: IAnalyticsStore,
+    private operationalStore: Knex,
```

Comment:
Knex -> Db :)
The less direct references we have to Knex in the code the better.